src/runtime_src/core/edge/user/aie/common_layer/adf_runtime_api.cpp

```diff
@@ -873,6 +873,19 @@ err_code dma_api::waitDMAChannelDone(int tileType, uint8_t column, uint8_t row,
     return err_code::ok;
 }
 
+err_code dma_api::updateBDAddress(int tileType, uint8_t column, uint8_t row, uint8_t bdId, uint64_t address)
```

Comment:
This code is a copy of aiecompiler code. Thatswhy didnt use XRT naming convention here,